_posts/2021-07-20-5.0.0-rs-release.md

```diff
@@ -0,0 +1,109 @@
+      35 // but Jorge is double counted
+-->
+
+The Rust Arrow implementation would not be possible without the wonderful work and support of our community, and the 5.0.0 release is no exception. It includes 161 commits from 34 individual contributors, many of them with their first contribution. Thank you all very much.
+
+# Arrow
+This release contains bug fixes, performance improvements, and a slimmer dependency stack when default features are disabled. Documentation has been improved and more examples have been added.
+
+Feature-wise, there is a [new kernel](https://github.com/apache/arrow-rs/pull/424) which lexicographically partitions points as well as usability enhancements for creating and manipulating Record Batches. Arrow Flight's API usability has [been improved](https://github.com/apache/arrow-rs/pull/377)
+
+We continue to leverage the Rust ecosystem to deliver reliable and performant code, and the Rust test suite now includes runs with the [MIRI checker](https://github.com/rust-lang/miri) (a sort of valgrind for Rust) for memory access violations.
+
+# Parquet
+The `parquet-derive` crate now automatically derives the required parquet schema, and the `parquet` crate had several bug fixes and enhancements.
+
+# More Frequent Releases
+Arrow releases major versions every three months. The Rust implementation has been experimenting with releasing minor version updates to speed the flow of new features and fixes. By implementing a new development process, as described in [A New Development Workflow for Arrow's Rust Implementation](https://arrow.apache.org/blog/2021/05/04/rust-dev-workflow/) we have successfully created 4 minor releases on the 4.x.x line every other week without any reports of breakage.
+
+You can always find the latest releases on crates.io: [`arrow`](https://crates.io/crates/arrow), [`parquet`](https://crates.io/crates/parquet), [`arrow-flight`](https://crates.io/crates/arrow-flight), and [`parquet-derive`](https://crates.io/crates/parquet-derive).
+
+# DataFusion & Ballista
+[DataFusion](https://docs.rs/datafusion/4.0.0/datafusion/) is an in-memory query engine with DataFrame and SQL APIs, built on top of Arrow. Ballista is a distributed compute platform. These projects are now in their [own repository](https://github.com/apache/arrow-datafusion), and are no longer released in lock-step with Arrow. Expect further news in this area soon.
+
+# Roadmap for 6.0.0 and Beyond
+Here are some of the initiatives that contributors are currently working on for future releases:
+
+* Improved performance of compute kernels
+* date/time/timestamp/interval compute kernels
+* A rewrite of the core of the array representation to remove the use of `unsafe` and make it faster and more secure -- see the [mailing list](https://lists.apache.org/thread.html/recac1f6dc982bab2923f8fb6992e2d4c927f46daff5f03ed6c4de19c%40%3Cdev.arrow.apache.org%3E) discussion for more details.
```

Comment:
@jorgecarleitao  is arrow2 ok to mention in the Roadmap section?

Reply:
I clarified that the roadmap involved preparing for this transition and left details vague